webapplication/aoi/models.py

```diff
@@ -69,6 +72,7 @@ class Component(models.Model):
     success = models.BooleanField(default=False, verbose_name='Validation succeeded')
     additional_parameter = models.CharField(max_length=50, null=True, blank=True, verbose_name='Additional parameter')
     run_on_gpu = models.BooleanField(default=True, verbose_name='Whether GPU is needed for a component to run')
+    sentinel_image_type = models.IntegerField(default=settings.SENTINEL_IMAGE_NONE_TYPE)
```

Comment:
You should compute and return a set of flags what kind of imagery is required in a view. No need to store this in db.